docs/Target-devices-and-platforms.md

```diff
@@ -0,0 +1,46 @@
+# Currently working devices:
```

Comment:
nit: can you rename the file to not use capitals?

Reply:
Yes